services/app/assets/js/widgets/components/FeedBackWidget.jsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import SlackFeedback, { themes } from 'react-slack-feedback';
+import { useSelector } from 'react-redux';
+
+import { currentUserNameSelector } from '../selectors/index';
+
+const FeedBackWidget = () => {
+  const currentUserName = useSelector(currentUserNameSelector);
+
+  const sendToServer = (payload, success, error) => fetch('/api/v1/feedback', {
+    method: 'POST',
+    headers: {
+      'x-csrf-token': window.csrf_token,
+    },
+    body: JSON.stringify(payload),
+  })
```

Comment:
```suggestion
 const sendToServer = (payload, success, error) => fetch('/api/v1/feedback', {
    method: 'POST',
    headers: {
      'Content-type': 'application/json',
      'x-csrf-token': window.csrf_token,
    },
    body: JSON.stringify(payload),
  })
```
Когда отправляем данные, всегда нужно обозначать какого они формата. Так же и при получении ответа, нам всегда приходит Сontent-type ответа, чтобы либа могла правильно интерпретировать данные.